selfdrive/loggerd/logger.h

```diff
@@ -15,18 +17,43 @@ const std::string LOG_ROOT = util::getenv_default("HOME", "/.comma/media/0/reald
 
 #define LOGGER_MAX_HANDLES 16
 
+class BZFile {
+ public:
+  BZFile(const char* path) {
+    assert(file = fopen(path, "wb"));
```

Comment:
Never put code inside an assert statement. If you compile with `-DNDEBUG` it would not open the file anymore.